.travis.yml

```diff
@@ -1,7 +1,28 @@
-language: node_js
-node_js:
-  - '8' # current stable
-script: yarn test --runInBand
+dist: xenial
+matrix:
+  include:
+  - name: "Node.js: 8"
+    language: node_js
+    node_js:
+    - '8' # current stable
+    cache: yarn
+    script: yarn test --runInBand
```

Comment:
This part is basically the same as before, but added `cache: yarn` for performance.

Reply:
Sounds good to me.